packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -10,10 +10,13 @@
  * governing permissions and limitations under the License.
  */
 
-import React, { useEffect, forwardRef } from 'react';
+import React, { useEffect, useState, useRef, createRef, forwardRef, cloneElement } from 'react';
 import classNames from 'classnames';
-import '@spectrum-css/tabs';
+import PropTypes from "prop-types";
```

Comment:
Can you pls run `yarn format` to fix formatting issues like the double quotes here ?

Reply:
This is done.